src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -2025,6 +2148,306 @@ private boolean checkParameters(String wardCode, String chargeCode, String disch
 		return false;
 	}
 	
+	private JComboBox<MedicalType> getJComboMedicalType() {
+		if (medicalTypeComboBox == null) {
+			medicalTypeComboBox = new JComboBox<MedicalType>();
+			try {
+				List<MedicalType> medicalTypes = medicalTypeManager.getMedicalType();	
+				MedicalType medicalType = new MedicalType(MessageBundle.getMessage("angal.common.all.txt"), MessageBundle.getMessage("angal.common.all.txt"));
+				medicalTypeComboBox.addItem(medicalType);
+				for (MedicalType medType: medicalTypes) {
+					medicalTypeComboBox.addItem(medType);
+				}
+			} catch (OHServiceException e) {
+				OHServiceExceptionUtil.showMessages(e);
+			}
+			
+			medicalTypeComboBox.addActionListener(actionEvent -> {
+				medicalTypeSelected = (MedicalType) medicalTypeComboBox.getSelectedItem();
+			});
+		}
+		return medicalTypeComboBox;
+	}
+	
+    private JRadioButton getAllRadioButton() {
```

Comment:
Fix indentation for all these buttons:
`getAllRadioButton`
`getMedicalWithNonZeroQuatityRadioButton`
`getMedicalWithMovementRadioButton`
`getCancelButton`

and leave an emtpy line after each `}`